lib/internal/bootstrap_node.js

```diff
@@ -412,6 +412,23 @@
 
       return caught;
     };
+
+    // This is a typed array for faster communication with JS.
+    const shouldAbortOnUncaughtToggle = process._shouldAbortOnUncaughtToggle;
```

Comment:
If we think there might be more of these kinds of config style toggles, it may be worthwhile creating a *single* TypedArray that can hold multiple that it exposed via `process.binding('config')` rather than introducing one per instance. For now, this is fine, I just would like to avoid having a growing number of one off TypedArray instances.